cmd/hubble/daemon_linux.go

```diff
@@ -66,6 +68,16 @@ var (
 				return nil, nil, fmt.Errorf("creating new controller-runtime manager: %w", err)
 			}
 
+			if err := ctrlManager.AddHealthzCheck("healthz", healthz.Ping); err != nil {
```

Comment:
Is healthz.Ping the HTTP Handler for the health check?

Reply:
Ping is like a placeholder function. It will always return 'true'

`// Ping returns true automatically when checked.
var Ping Checker = func(_ *http.Request) error { return nil }
`

https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/healthz/healthz.go

Reply:
Updated